pom.xml

```diff
@@ -62,6 +62,7 @@
         <dep.nexus-staging-plugin.version>1.6.8</dep.nexus-staging-plugin.version>
         <dep.asm.version>6.2.1</dep.asm.version>
         <dep.gcs.version>1.9.17</dep.gcs.version>
+        <dep.alluxio.version>2.1.0</dep.alluxio.version>
```

Comment:
should this be 2.1.1 at least? 2.1.0 did not have support for prestodb right?

Reply:
AFAIK there are no changes made to Alluxio that force the need for 2.1.1